tests/commands/test_cmd_config.py

```diff
@@ -0,0 +1,73 @@
+# third-party imports
+from click.testing import CliRunner
+
+# local imports
+from shpkpr.cli import cli
+
+
+def test_no_args():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['config'])
+
+    assert result.exit_code == 0
```

Comment:
This is an odd one. The config command is actually a group which contains 3 subcommands. The default behaviour is that it does nothing and exits. It's reasonable to expect this to act like the other top level commands and throw a help screen and exit 2. I'll address in a follow up PR.